### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by jiaaihhy** [zzh-tech/ESTRNN#27]
dear author:
I have a question that training on gopro, have 2 blur folder, that blur and blur_gamma, I want to know you used is blur or blur_gamma? thankyou

**Comment by zzh-tech** [comment]
blur_gamma

**Comment by lyw3074** [comment]
dear zzh:
Whether the image resolution of the GOPRO dataset is downsampled from 1280*720 to 960*540 like IFIRNN？ thank you very much！

**Comment by zzh-tech** [comment]
Yes

**Comment by Iverson-zq** [comment]
> dear author:
> I have a question that training on gopro, have 2 blur folder, that blur and blur_gamma, I want to know you used is blur or blur_gamma? thankyou

Hi ,bro，Have you trained models on GOPRO datasets?I have a few questions for you ,When I downloaded the GOPRO_DS ,I had some problems during the training.

RuntimeError: There were no tensor arguments to this function (e.g., you passed an empty list of Tensors), but no fallback function is registered for schema aten::_cat.  This usually means that this function requires a non-empty list of Tensors.  Available functions are [CPU, CUDA, QuantizedCPU, Autograd, Profiler, Tracer, Autocast]

I just changed the batch_size=8 to batch_size=4.
